documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -55,7 +55,7 @@ This is a basic configuration example, which is also located in the sources of t
 steps:
   mtaBuild:
     buildTarget: 'NEO'
-    mtaJarLocation: '/opt/sap/mta.jar'
```

Comment:
Did you remove this line because we are in a Docker only scenario?

Reply:
yes, and the default setting is correct. This one did not work.